### DISCUSSION TITLE
Seeking Automated Background Editing and Color Adjustment Features

### DISCUSSION THREAD

**Discussion by BooleanBarista** [T8RIN/ImageToolbox#1064] [Q&A]
Hello,

I’m currently using ImageToolbox for editing photos of objects placed centrally in a small photo box with a white background. Occasionally, the edges of the photo box or slight shadows become visible in the final images. I need to remove these without changing the image format or aspect ratio, and simple cropping isn’t an option.

I’m looking for features within ImageToolbox that would allow me to:

- Automatically detect and remove the background, replacing it with either pure white or the actual color of the photo box’s background, which is a light but slightly grayish white.
- Use a pipette tool to pick up the background color and use it to paint over the unwanted shadows.

Additionally, I’ve encountered an issue with the “Remove Background” feature where the background turns black upon saving the image, and I haven’t found an option to change this color.

Could you please advise if these functionalities are available or if there are workarounds within the app? Any guidance on how to maintain the original color while removing the background would be greatly appreciated.

Thank you for your assistance!
BooleanBarista

**Comment by T8RIN** [comment]
Black color can appear if you save in format without transparency, btw i think in newer versions all is ok👀
